source/deploy/mobile-app-changelog.md

```diff
@@ -2,12 +2,52 @@
 
 Latest Mattermost Mobile Apps releases:
 
-- [2.0.0 Release](#release)
-- [1.55.1 Release](#id1)
-- [1.55.0 Release](#id4)
-- [1.54.0 Release](#id8)
-- [1.53.0 Release](#id13)
-- [1.52.0 Release](#id17)
+- [2.0.1 Release](#release)
+- [2.0.0 Release](#id1)
+- [1.55.1 Release](#id4)
+- [1.55.0 Release](#id8)
+- [1.54.0 Release](#id13)
+- [1.53.0 Release](#id18)
+
+## 2.0.1 Release
+- Release Date: 
```

Comment:
The release date will be added after the 'release' is done. Correct?

Reply:
Yes, I mentioned that in the PR description.